### PR TITLE
Refs #32694 -- Clarified when colorama requirement is needed in Windows how-to.

### PR DESCRIPTION
**Pull request by carltongibson**

### CODE REVIEW

File: docs/howto/windows.txt [hunk 1]
```diff
@@ -96,12 +96,13 @@ Colored terminal output
 
 .. versionadded:: 3.2
 
-A quality-of-life feature is to output colored (rather than monochrome) output
-on the terminal. This should work both on CMD and PowerShell. If for some
-reason this needs to be disabled, set the environmental variable
+A quality-of-life feature adds colored (rather than monochrome) output to the
+terminal. In modern terminals should work for both CMD and PowerShell. If for
```
Comment (smithdc1) [inline docs/howto/windows.txt:100]: typo near "terminals" and "should"

File: docs/howto/windows.txt [hunk 2]
```diff
@@ -96,12 +96,13 @@ Colored terminal output
...
+some reason this needs to be disabled, set the environmental variable
 :envvar:`DJANGO_COLORS` to ``nocolor``.
 
-To enable this, colorama_ must be installed::
+On older Windows versions, or legacy terminals, colorama_ must be installed to
```
Comment (smithdc1) [inline docs/howto/windows.txt:104]: What do you mean by "legacy terminals"? I'd have called CMD legacy?

**Review by carltongibson — COMMENTED** [review]
Essentially Powershell/VS Code/Windows Terminal &co are "modern" — all run with colour (the latter two running CMD with colour).
Legacy (on Windows 10) is the _Command Prompt_ application, which runs CMD, but no colors.

**Review by felixxm — APPROVED** [review]
@carltongibson Thanks :+1:

File: docs/howto/windows.txt [hunk 3]
```diff
@@ -96,12 +96,13 @@ Colored terminal output
 
 .. versionadded:: 3.2
 
-A quality-of-life feature is to output colored (rather than monochrome) output
-on the terminal. This should work both on CMD and PowerShell. If for some
-reason this needs to be disabled, set the environmental variable
+A quality-of-life feature adds colored (rather than monochrome) output to the
+terminal. In modern terminals should this work for both CMD and PowerShell. If
```
Comment (pope1ni) [inline docs/howto/windows.txt:100]:
```suggestion
terminal. In modern terminals this should work for both CMD and PowerShell. If
```

**Review by pope1ni — CHANGES_REQUESTED** [review]

File: docs/howto/windows.txt (hunk 3 above)
Comment (carltongibson) [reply docs/howto/windows.txt]: Thanks Nick. 🤯

**Pull request merged**